raiden/smart_contracts/NettingChannelLibrary.sol

```diff
@@ -43,26 +43,22 @@ library NettingChannelLibrary {
 
 
     modifier notSettledButClosed(Data storage self) {
-        if (self.settled > 0 || self.closed == 0)
-            throw;
+        assert(self.settled <= 0 && self.closed > 0);
         _;
     }
 
     modifier stillTimeout(Data storage self) {
-        if (self.closed + self.settle_timeout < block.number)
-            throw;
+        assert(self.closed + self.settle_timeout >= block.number);
         _;
     }
 
     modifier timeoutOver(Data storage self) {
-        if (self.closed + self.settle_timeout > block.number)
-            throw;
+        assert(self.closed + self.settle_timeout <= block.number);
         _;
     }
 
     modifier channelSettled(Data storage self) {
-        if (self.settled == 0)
-            throw;
+        assert(self.settled != 0);
```

Comment:
Should be a `require`. All these are states of the contract which can be enountered, not invalid states that can never be reached.